dist/profile/manifests/updates.pp

```diff
@@ -0,0 +1,23 @@
+#
+# Defines a updates.jenkins.io server for serving updatesite, aims to deal with volume management
+#
+class profile::updates() {
+  include ::stdlib
+  # volume configuration is in hiera
+  include ::lvm
```

Comment:
```suggestion
  include stdlib
  # volume configuration is in hiera
  include lvm
```

Since https://github.com/jenkins-infra/jenkins-infra/pull/2255, we have to use the Puppet v6 syntax (tip: install the Puppet Development Kit plugin on vscode to help you auto-format files)

Reply:
is this the good one ? : [puppet-vscode](https://marketplace.visualstudio.com/items?itemName=puppet.puppet-vscode)

Reply:
Yep, that is the one I'm using

Reply:
so I had it already and no detection for those "::"